youtubefunction.py

```diff
@@ -18,11 +19,12 @@ def getYouTube(self, queryy) :
             
             ytLinkList = []
             
+            #find links to videos
             for link in soup.find_all('a', href= re.compile('watch')) :
                 if('https://www.youtube.com/' + link['href'] not in ytLinkList) :
                     ytLinkList.append('https://www.youtube.com/' + link['href'])
                     
-            return ytLinkList
+            return ytLinkList[:10]

```

Comment:
Why only the first 10? Can you add a comment that talks about your choice to only get the top 10

Reply:
I've added a size parameter that's adjusted in maindata.py file. I had originally set it to 10 for consistency between the functions, but I've changed that to a variable number.